### PR TITLE
models with validations are done

### PR DESCRIPTION
**Pull request by galia1902**

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Game < ApplicationRecord
+  belongs_to :user
+  has_many :rounds
+  validates :game_mode, presence: true
+  validates :user_id, presence: true
+  validates :score, presence: true
```
Comment (Grokepau) [inline app/models/game.rb]: no validation for score because we get the score after we started the game

File: app/models/game.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class Game < ApplicationRecord
+  belongs_to :user
+  has_many :rounds
+  validates :game_mode, presence: true
```
Comment (Grokepau) [inline app/models/game.rb]: can you also check for specific game options?
for example game mode can only be:
-Try Out
-MQA
-Sortable

File: db/migrate/20200223191535_create_rounds.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class CreateRounds < ActiveRecord::Migration[5.2]
+  def change
+    create_table :rounds do |t|
+      t.datetime :duration
```
Comment (Grokepau) [inline db/migrate/20200223191535_create_rounds.rb]: sure we shouldn't just use integers?

File: app/models/game.rb (hunk 1 above)
Comment (galia1902) [reply app/models/game.rb]: it should have a default value at creation of the class instance. I'm gonna add it now

File: app/models/game.rb (hunk 2 above)
Comment (galia1902) [reply app/models/game.rb]: added

File: db/migrate/20200223191535_create_rounds.rb (hunk 3 above)
Comment (galia1902) [reply db/migrate/20200223191535_create_rounds.rb]: not sure. what do you think?

**Pull request merged**